libavcodec/cbs_h266_syntax_template.c

```diff
@@ -3704,6 +3704,54 @@ SEI_FUNC(sei_picture_timing, (CodedBitstreamContext *ctx, RWContext *rw,
     return 0;
 }
 
+SEI_FUNC(sei_decoding_unit_info, (CodedBitstreamContext *ctx, RWContext *rw,
+                                  H266RawSEIDecodingUnitInfo *current,
+                                  SEIMessageState *sei))
+{
+    CodedBitstreamH266Context *h266 = ctx->priv_data;
+    H266RawSEIBufferingPeriod *bp   = h266->bp;
+
+    int temporal_id = h266->temporal_id;
+    int err, i;
+
+    HEADER("DU information");
+
+    if (!bp) {
+        av_log(ctx->log_ctx, AV_LOG_WARNING,
+            "Skipped DU information SEI message for no available buffer period.\n");
+        return AVERROR_INVALIDDATA;
+    }
+
+    ue(dui_decoding_unit_idx, 0, 0xffffffff);
```

Comment:
The value of dui_decoding_unit_idx shall be in the range of 0 to PicSizeInCtbsY − 1, inclusive